package.json

```diff
@@ -13,13 +13,15 @@
         "@typescript-eslint/eslint-plugin": "^2.28.0",
         "@typescript-eslint/parser": "^2.28.0",
         "eslint": "^6.8.0",
-        "eslint-config-airbnb-base": "^14.1.0",
         "eslint-config-prettier": "^6.10.1",
         "eslint-plugin-prettier": "^3.1.3",
-        "prettier": "^2.0.4"
+        "prettier": "2.0.4"
```

Comment:
I think we can probably go with ^2.0.4 here, unless you know why we can't.

Reply:
Prettier documentation recommended using exact: "We recommend pinning an exact version of prettier in your package.json as we introduce stylistic changes in patch releases."
But if you think we should go with ^2.0.4, I'm fine with that.

Reply:
Ingrid and I had an offline conversation about this. Indeed uses ^ in its package.json files. We considered going with ~2.0.4 to allow minor patches (such as security fixes), but not minor version bumps. Since minor version bumps are *supposed* to be backward compatible, we decided to go with ^2.0.4 and we can just change things if the prettier folks break the rules.